paper.tex

```diff
@@ -724,77 +724,71 @@ \section*{Genome ARGs}
 
 
 \section*{Ancestry resolution}\label{Ancestry_resolution}
-% This is a rough first pass. Will clarify terminology later.
 % Need to introduce Hudson's algorithm as concerned with simulating the process of
 % the coalescent with recombination as efficiently as possible, and properly merge in
 % with this section.
 
-The definition of an ARG as a data structure in terms of
-individuals' monoploid genomes
-and their direct patterns of inheritance allows us to describe arbitrarily
-complex relatedness structures, free from the limitations imposed by
-coalescent approximations.
-An ARG is an intrinsically retrospective
-structure, defined with respect to the ancestry of
-a set of sampled genomes.
-The graph structure (the nodes and their connectivity through edges)
-captures a great deal of information about this ancestry,
-but on its own it cannot tell us about the precise relationships between
-the samples along the genome: do to this, we must construct the
-local (``marginal'') trees.
-
-We can define a sample-resolved gARG as a tuple $(S, E)$
-where $S \subset \mathbb{N}$ is a set of sample nodes and
-$E$ is a set of edges. Each element of $E$
-is a tuple $(c, p, I)$ such that $c, p \in \mathbb{N}$,
-$c$ is the child node, $p$ is the parent node and $I$ is the set of
-disjoint genomic intervals $(\ell, r]$
-over which genome $c$ inherits from $p$, and there is a path from
-$c$ to some non-empty subset of $S$ through the local trees.
-% Should probably define how we build the local trees somewhere.
-
+The edge annotations we have described in Fig 1 and 2 show the regions of genome
+that have been passed from a parent to its child in their entirety. In effect, all edges
+span the entire length of the genome, apart from those that connect two parents
+to a child in which they recombine. In this form of edge annotation, which we call
+``entire edge encoding'', it is inefficient to extract the local trees from the ARG structure.
+
+There is, however, an alternative edge annotation scheme which allows efficient
+storage and extraction of the local trees. It is based on the fact that the ARG
+structure is inherently retrospective, defined as the ancestry of a set of \emph{sample}
+nodes (such as \textsf{a}, \textsf{b}, and \textsf{c} in the examples above). We define a
+``sample-resolved edge encoding'' to be a representation where edge spans have been
+reduced in extent so that they represent only those parts of the genome that have
+been inherited by the samples. This also allows the removal of nodes which are not
+genetically ancestral to the samples.
 
 \begin{figure}
 \centering
 \includegraphics[width=\textwidth]{illustrations/ancestry-resolution}
 \caption{\label{fig-ancestry-resolution}
 The \citet[][fig. 1]{wiuf1999recombination} example graph as a gARG, with contrasting
 edge annotation schemes.
-(A) Edge annotations that reflect immediate inheritance, i.e. transmission of
```

Comment:
I think these changes are worse actually "encoding" is a term we want to avoid where we can. We're recording different information here, not using a different way to write down the same information.

Reply:
Good point. Let's discuss in our meeting later today.